specs/core/0_beacon-chain.md

```diff
@@ -1,18 +1,61 @@
 # Ethereum 2.0 Phase 0 -- The Beacon Chain
 
-###### tags: `spec`, `eth2.0`, `casper`, `sharding`, `beacon`
+**tags: `spec`, `eth2.0`, `casper`, `sharding`, `beacon`**
```

Comment:
The tags were originally written for better sorting experience in Hackmd, they can now be moved to Github repository tags, which would looks like the following example image.
<img width="678" alt="screen_shot_2018-11-28_at_6_47_41_pm" src="https://user-images.githubusercontent.com/3391420/49146940-40791880-f33e-11e8-91c4-f971e00da7b0.png">